common/provisioner_utils.go

```diff
@@ -233,7 +230,7 @@ func CreateLocalPV(
 		return nil
 	})
 	if opRes != controllerutil.OperationResultNone {
-		pvLogger.Info("pv changed", "operation", opRes)
+		klog.Infof("PV changed, name = %s, status = %s", pvName, opRes)
```

Comment:
Could we use `InfoS` in these places?

Reply:
klog doesn't have any structured logging functions that I could find... at least not this version of it. Am I missing something obvious?

Reply:
Ah, that was added in klog v2 it looks like, and LSO is still on v1:
```
$ grep klog go.mod 
        k8s.io/klog v1.0.0
```

Reply:
hmm okay. Lets handle this later.

Reply:
Hold on, I made the changes, but need to retest.

Reply:
Pushed.